src/Microsoft.Sbom.Api/Config/ConfigSanitizer.cs

```diff
@@ -25,6 +26,13 @@ public class ConfigSanitizer
         private readonly IFileSystemUtils fileSystemUtils;
         private readonly IAssemblyConfig assemblyConfig;
 
+        internal static string SBOMToolVersion => VersionValue.Value;
+
+        private static readonly Lazy<string> VersionValue = new Lazy<string>(() =>
+        {
```

Comment:
I think you can omit the `return` and braces here and just let this be a [expression lambda](https://learn.microsoft.com/en-us/dotnet/csharp/language-reference/operators/lambda-expressions#expression-lambdas).